build/run.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
```

Comment:
You can check the required variables here just like we do it in https://github.com/cfengine/masterfiles/blob/master/templates/federated_reporting/import.sh#L12